Core/Utilities/ElasticSearchDB.py

```diff
@@ -0,0 +1,279 @@
+########################################################################
+# $Id: $
+########################################################################
+
+"""
+This class a wrapper around elasticsearch-py. It is used to query
+Elasticsearch database.
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                      import gLogger, S_OK, S_ERROR
+from elasticsearch              import Elasticsearch
+from elasticsearch_dsl          import Search, Q, A
+from elasticsearch.exceptions   import ConnectionError, TransportError, NotFoundError
+from elasticsearch.helpers      import BulkIndexError
+from elasticsearch              import helpers
+from datetime                   import datetime
+from DIRAC.Core.Utilities       import Time
+
+class ElasticSearchDB( object ):
+  
+  """
+  .. class:: ElasticSearchDB
+
+  :param str url: the url to the database for example: el.cern.ch:9200
+  :param str gDebugFile: is used to save the debug information to a file
+  :param int timeout the default time out to Elasticsearch
+  """
+  __chunk_size = 1000
+  __url = ""
+  __timeout = 120
+  ########################################################################
+  def __init__( self, host, port, debug = False ):
+    """ c'tor
+    :param self: self reference
+    :param str host: name of the database for example: MonitoringDB
+    :param str port: The full name of the database for example: 'Monitoring/MonitoringDB'
+    :param bool debug: save the debug information to a file   
+    """
+    global gDebugFile
+    
+    if not hasattr( self, 'log' ):
+      self.log = gLogger.getSubLogger( 'ElasticSearch' )
+    self.logger = self.log
+    
+    self.__url = "http://%s:%s" % ( host, port )
+        
+    if debug:
+      try:
+        gDebugFile = open( "%s.debug.log" % self.__dbName, "w" )
```

Comment:
yes. It is in standard location. This is used to separate the DB logs from the service log. I have removed it.